packages/app-project/src/shared/components/Stat/components/AnimatedNumber/AnimatedNumber.js

```diff
@@ -9,14 +9,15 @@ class AnimatedNumber extends Component {
   }
 
   componentDidMount () {
-    this.animateValue()
+    this.animateValue(0)
   }
 
   componentDidUpdate (prevProps) {
     this.animateValue(prevProps.value)
   }
 
-  animateValue (prevValue = 0) {
+  animateValue (prevValue) {
+    const self = this
```

Comment:
Non-blocking, but why did you need to do this?

Reply:
Woops, not sure why it was marked resolved already. Anyway, it turns out `calledWithArgs` doesn't pick up default arguments